app/Console/Commands/CreateMasterVoucherLogReport.php

```diff
@@ -218,17 +218,19 @@ public function handle()
                     unset($chunk[$k]);
                     continue;
                   }
-                  if (!is_null($voucher['Date Trader Recorded Voucher'])
-                        && (strtotime($voucher['Date Trader Recorded Voucher']) < strtotime('2020-01-01'))
-                      ) {
-                    unset($chunk[$k]);
-                    continue;
+                  if (!is_null($voucher['Date Trader Recorded Voucher'])) {
+                        $date = \DateTime::createFromFormat('d/m/Y', $voucher['Date Trader Recorded Voucher']);
```

Comment:
If date-maths turns out to be slow, we could be cheeky and a substring or regexp on compare on `201[0-9]-`